packages/inter-protocol/src/proposals/replaceElectorate.js

```diff
@@ -214,6 +228,23 @@ const startNewEconomicCommittee = async (
   trace(`committeeName ${committeeName}`);
   trace(`committeeSize ${committeeSize}`);
 
+  // if retiredContractInstances doesn't exist, create it.
+  const contractInstanceMap = makeScalarBigMapStore(
+    'retiredContractInstances',
+    { durable: true },
+  );
+  retiredInstanceWriter.resolve(contractInstanceMap);
+
+  // get the actual retiredContractInstances
+  const retiredInstancesWriter = await retiredInstancesP;
```

Comment:
nit: having both `retiredInstanceWriter` and `retiredInstancesWriter` (it took me a few minutes to spot the extra `s` in the second one) is confusing. Could the second one simply be called `retiredInstances`?

Reply:
yes,  definitelly. I don't know how that happened. The latter is not even a writer.